lms/djangoapps/certificates/management/commands/cert_whitelist.py

```diff
@@ -57,6 +57,18 @@ def handle(self, *args, **options):
         if not course_id:
             raise CommandError("You must specify a course-id")
 
+        def update_user_whitelist(username, add=True):
```

Comment:
I think we should do that as a separate small refactor, but looks like the right thing to do.